Include notification for new directory structure

salameer opened the pull request:
Adding description to the new upcoming changes to the repos folder structure

This checklist is used to make sure that common issues in a pull request are addressed. This will expedite the process of getting your pull request merged and avoid extra work on your part to fix issues discovered during the review process. 

### PR information
- [X ] The title of the PR is clear and informative.
- [X ] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For information on cleaning up the commits in your pull request, [see this page](https://github.com/Azure/azure-powershell/blob/dev/documentation/cleaning-up-commits.md).
- [X ] Except for special cases involving multiple contributors, the PR is started from a fork of the main repository, not a branch.
- [X ] If applicable, the PR references the bug/issue that it fixes.
- [X ] Swagger files are correctly named (e.g. the `api-version` in the path should match the `api-version` in the spec).

### Quality of Swagger
- [ X] I have read the [contribution guidelines](https://github.com/Azure/azure-rest-api-specs/blob/master/.github/CONTRIBUTING.md).
- [X ] My spec meets the review criteria:
  - [X ] The spec conforms to the [Swagger 2.0 specification](https://github.com/OAI/OpenAPI-Specification/blob/master/versions/2.0.md).
  - [ X] The spec follows the guidelines described in the [Swagger checklist](../documentation/swagger-checklist.md) document.
  - [X] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.

salameer committed:
Include notification for new directory structure

Adding description to the new upcoming changes to the repos folder structure

kirthik commented:
Could you please take a look at the comments in https://github.com/Azure/azure-rest-api-specs/pull/1360 and address them here? Not sure why that PR got closed and new one was opened ...

salameer committed:
Update README.md

salameer committed:
Fixed the broken  Directory structure

Fixed the structure

salameer committed:
Updated Tag Value

updated Tag values to "package-2017-03"

I hope this is what you meant cause honestly I don't have a lot of context on the decision or pattern for this value :)

salameer commented:
Captain @kirthik 

Your comments have been addressed in my latest commits, please let me know if there's more to be changed here

Thanks and God Speed LOL

vishrutshah commented on README.md:
````diff
@@ -57,10 +57,61 @@ The structure should appear like so:
+ 
+```bash
+.
+├──automation
````
Not sure whether we intentionally skipped the top node `specification` or not https://github.com/Azure/azure-rest-api-specs/tree/reorg. Other than that LGTM! 👍

salameer replied:
hmmmm, good point let me add it

salameer committed:
Added root folder

Added root folder "Specification"

AutorestCI commented:
No modification for AutorestCI/azure-sdk-for-python

AutorestCI commented:
No modification for AutorestCI/azure-sdk-for-ruby

AutorestCI commented:
No modification for AutorestCI/azure-sdk-for-node
